### ISSUE TITLE
Incompatible libc and libc++ on Linux (Ubuntu 20.04)

### ISSUE THREAD

**Issue by pierreguillot** [Tracktion/pluginval#84]
I'm using the latest pluginval binaries on my CI and since the 1.0.0 update, the libc and libc++ are incompatible:
```
.../pluginval: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.35' not found (required by .../pluginval)
.../pluginval: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.30' not found (required by .../pluginval)
.../pluginval: /lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.29' not found (required by .../pluginval)
.../pluginval: /lib/x86_64-linux-gnu/libstdc++.so.6: version `CXXABI_1.3.13' not found (required by .../pluginval)
.../pluginval: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by .../pluginval)
.../pluginval: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by .../pluginval)
.../pluginval: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by .../pluginval)
```
I'm on Ubuntu 20.04 and it used to work fine with pluginval v0.3.0. I guess pluginval 1.0.0 is compiled against newer versions of libc and libc++ which the ABI is not compatible with the libs of my system. Is this change intended? I would like to avoid compiling my own version of pluginval or updating the version of the OS 😇.

**Comment by drowaudio** [comment]
Do you happen to know if we static link libstdc++ would it work?
Doesn't libc have a stable ABI? Or will the library loader still complain?

**Comment by pierreguillot** [comment]
I don't know, sorry, I'm not a Linux specialist... I guess it depends on the use of "new" symbols 🤷 [API/ABI changes review for glibc](https://abi-laboratory.pro/?view=timeline&l=glibc).

**Comment by drowaudio** [comment]
Does this work for you?
[pluginval_Linux.zip.zip](https://github.com/Tracktion/pluginval/files/9300562/pluginval_Linux.zip.zip)

**Comment by pierreguillot** [comment]
Yes, this one seems to work! Thank you :)

**Comment by drowaudio** [comment]
Fixed with 1.0.1 here: https://github.com/Tracktion/pluginval/releases/tag/v1.0.1
